azure-pipelines.yml

```
@@ -83,6 +79,11 @@ jobs:
       Python38:
         python.version: '3.8'
         SKLEARN_VERSION: "*"
+      Scikit_1:
```

There are many scikits != sklearn https://www.scipy.org/scikits.html,
```suggestion
      Py39_sklearn1:
```